src/test/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/refreshanalyzer/RefreshSearchAnalyzerActionIT.kt

```diff
@@ -0,0 +1,270 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.indexmanagement.refreshanalyzer
+
+import com.amazon.opendistroforelasticsearch.indexmanagement.IndexManagementPlugin
+import com.amazon.opendistroforelasticsearch.indexmanagement.IndexManagementRestTestCase
+import org.elasticsearch.client.Request
+import org.elasticsearch.common.io.Streams
+import org.elasticsearch.common.settings.Settings
+import org.elasticsearch.common.xcontent.XContentType
+import java.io.InputStreamReader
+import java.nio.charset.Charset
+import java.nio.charset.StandardCharsets
+import java.nio.file.Files
+
+class RefreshSearchAnalyzerActionIT : IndexManagementRestTestCase() {
+    fun `test index time analyzer`() {
+        val buildDir = System.getProperty("buildDir")
+        val numNodes = System.getProperty("cluster.number_of_nodes", "1").toInt()
+        val indexName = "testindex"
+
+        for (i in 0 until numNodes) {
+            writeToFile("$buildDir/testclusters/integTest-$i/config/pacman_synonyms.txt", "hello, hola")
+        }
+
+        val settings: Settings = Settings.builder()
+                .loadFromSource(getIndexAnalyzerSettings(), XContentType.JSON)
+                .build()
+        createIndex(indexName, settings, getAnalyzerMapping())
+        ingestData(indexName)
+        Thread.sleep(1000) // wait for refresh_interval
```

Comment:
Can we remove this and just wrap the next assertion in a waitFor block

Reply:
sure